apollo-client/src/main/java/com/ctrip/framework/apollo/Config.java

```diff
@@ -18,6 +18,13 @@
   public String getProperty(String key, String defaultValue);
 
   /**
+   * Return the PlainText, such as xml. yml
+   * @param defaultValue
+   * @return
+   */
+  String getPlainTextConfig(String defaultValue);
```

Comment:
why adding this new api?

Reply:
easy to find the config, such as xml, yml...
the config divide two group, properties and others. so i add a new api to get non-properties config.

Reply:
I would say to not add this api.
Because you could use ConfigService.getConfigFile() to easily get a config file object, but not from ConfigService.getConfig(), as the namespace name contains the namespace file extension.

Reply:
accept you point